components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIStoreHostObject.java

```diff
@@ -2835,7 +2835,7 @@ private static void addAPIObj(SubscribedAPI subscribedAPI, NativeArray apisArray
             apiObj.put("hasMultipleEndpoints", apiObj, String.valueOf(api.getSandboxUrl() != null));
             apisArray.put(apisArray.getIds().length, apisArray, apiObj);
         } catch (APIManagementException e) {
-            handleException("Error while obtaining application metadata", e);
+            log.error("Error while obtaining application metadata", e);
```

Comment:
according to the above mentioned jira if registry artifact of api is removed frrom the management console and go to access store will occur this exception as  this exception throws api store couldn't show the apis if registry corruption happen